packages/odyssey/src/scss/components/_toast.scss

```diff
@@ -0,0 +1,138 @@
+// Copyright 2018-Present Okta, Inc.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+.ods-toast {
+  display: grid;
+  position: relative;
+  grid-gap: 0 $spacing-s;
+  grid-template-columns: min-content 1fr min-content;
+  align-items: center;
+  max-width: $max-line-length;
+  padding: $spacing-m $spacing-s;
+  animation-name: ods-toast-in, ods-toast-out;
+  animation-duration: 300ms, 1000ms;
+  animation-delay: 0s, 5300ms;
+  border: 0;
+  border-radius: $base-border-radius;
+  background-color: cv('blue', '900');
+  box-shadow: 0 8px 12px rgba(cv('gray', '900'), 0.12);
```

Comment:
Should we use opacity to get the desired color for shadow? Are there other existing colors we can use out of the box or play around with the spread?

Reply:
@joejezowski-okta any thoughts here? No strong opinions from me (for now 😇). 

Unless we want true transparency on underlying elements, it would be nice to use a standard color from the set.

Reply:
I think in the case of toast, which is the first of many "overlay" class of components, we do want true transparency on background content. Solid colors will look incorrect in that case.

Reply:
Again, nitpicking here and not going to block the PR on this, but you would be getting "transparency" for free with just with box shadow and spread, right?

Reply:
Oh, I see what you are telling now. Transparency makes sense here - since we want to see the content below.

Reply:
I've added this to our shadow standardization ticket: https://github.com/okta/odyssey/issues/95